cmds/wifi/wifiServer.go

```diff
@@ -43,29 +42,41 @@ function replaceWithConnecting(elem) {
     connectingTxt = document.createTextNode("Connecting...")
     elem.style.display = "none"
     elem.parentNode.appendChild(connectingTxt);
+	disableOtherButton(elem)
+}
+
+function sendRefresh(elem) {
+	elem.setAttribute("disabled", "true")
+	elem.setAttribute("value","Refreshing")
+	disableOtherButton(elem)
+	fetch("http://localhost:8080/refresh").then(_ => window.location.reload())
+}
+
+function disableOtherButton(elem) {
     btns = document.getElementsByClassName("btn");
-    var i;
+	var i;
     for (i = 0; i < btns.length; i++) {
     	if (btns[i] === elem) {
     		continue;
```

Comment:
Consider https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of if possible, it is less errorprone.

Reply:
ok, I'll update this